cmd.c

```diff
@@ -897,6 +882,75 @@ void cmd_syspage(char *s)
 }
 
 
+void cmd_phfs(char *s)
+{
+	u16 argsc = 0;
+	unsigned int i, major, minor, pos = 0;
+	char args[MAX_CMD_ARGS_NB][LINESZ + 1];
+
+	for (i = 0; argsc < MAX_CMD_ARGS_NB; ++i) {
+		cmd_skipblanks(s, &pos, ". \t");
+		if (cmd_getnext(s, &pos, ". \t", NULL, args[i], sizeof(args[i])) == NULL || *args[i] == 0)
+			break;
+		argsc++;
+	}
+
+	if (argsc < 3 || argsc > 4) {
+		plostd_printf(ATTR_ERROR, "\nLack of obligatory arguments!!\n");
+		return;
+	}
+
+	/* Get major/minor */
+	major = plostd_ahtoi(args[1]);
+	minor = plostd_ahtoi(args[2]);
+
+	if (phfs_regDev(args[0], major, minor, (argsc == 3) ? NULL : args[3]) < 0)
+		plostd_printf(ATTR_ERROR, "\n%s is not registered!\n", args[0]);
+}
+
+
+void cmd_devs(char *s)
+{
+	u16 argsc = 0;
+	char args[MAX_CMD_ARGS_NB][LINESZ + 1];
+
+	/* Parse command arguments */
+	if (cmd_parseArgs(s, args, &argsc) > 0) {
```

Comment:
If it will not been expanded:
```suggestion
	unsigned int pos = 0;

	cmd_skipblanks(s, &p, DEFAULT_BLANKS);
	s += p;

	if (*s) {
```

Reply:
Done